src/patternfly/components/FormControl/form-control.hbs

```diff
@@ -1,15 +1,14 @@
-<{{ controlType }} class="pf-c-form-control{{#if pf-c-form-control--modifier}} {{pf-c-form-control--modifier}}{{/if}}"
-{{#if type}} type="{{type}}"{{/if}}
-{{#if placeholder}} placeholder="{{placeholder}}"{{/if}}
-{{#if value}} value="{{value}}"{{/if}}
-{{#if name}} name="{{name}}"{{/if}}
-{{#if readonly}} readonly{{/if}}
-{{#if aria-describedby}} aria-describedby="{{aria-describedby}}"{{/if}}
-{{#if aria-label}} aria-label="{{aria-label}}" {{/if}}
-{{#if aria-errormessage}} aria-errormessage="{{aria-errormessage}}"{{/if}}
-{{#if required}}required aria-required="true"{{/if}}
-{{#if aria-invalid}}aria-invalid="true"{{/if}}
-{{#if disabled}} disabled{{/if}}
-{{#if id}}id="{{id}}"{{/if}}
-{{#if multiple}} multiple="true"{{/if}}
+<{{ controlType }} class="pf-c-form-control{{#if form-control--modifier}} {{form-control--modifier}}{{/if}}"
+  {{#if form-control--attribute}}
+    {{{form-control--attribute}}}
+	{{/if}}
+	{{#if disabled}} 
```

Comment:
I thnk we can simplify this further by removing this conditionals and just having the disabled,readonly, and required attributes in the form-control--attribute rather than the disabled=true, required=true, and readonly=true. Sorry I should have been more clear about this one.

Reply:
removed :)